examples/catvdog/klio-job.yaml.example

```diff
@@ -28,7 +29,12 @@ job_config:
     inputs:
       - type: gcs
         location: gs://$GCP_PROJECT-input/catvdog
+        file_suffix: .jpg
     outputs:
       - type: gcs
         location: gs://$GCP_PROJECT-output/catvdog
+        file_suffix: .jpg
+        # as of klio-exec 0.2.0, multiple outputs aren't supported
+        # TODO: update me once it is supported!
```

Comment:
is this just like a FYI or would this example job actually have multiple outputs in the future?

Reply:
It does have a type of multiple outputs - output could either be uploaded to `<gcs_bucket>/dog` or `<gcs_bucket>/cat`.

Reply:
I could re-work this a little bit so multiple outputs aren't needed 🤔

Reply:
actually, it isn't that easy to re-work - but I can do some custom existence checks (which would have the `skip_klio_existence_check` still set to `True` so not much different).